boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -32,13 +35,13 @@ if [ ! -d "${SAAS_OPERATOR_DIR}/.git" ] ; then
 fi
 
 cat <<EOF > $DOCKERFILE_REGISTRY
-FROM quay.io/openshift/origin-operator-registry:latest
+FROM quay.io/openshift/origin-operator-registry:4.8.0
```

Comment:
:pray: - though this is the release that OLM reported pushing a regression. I tested `latest` it was ok but I also tested `4.7` which was fine and that matches our OCP version, go with that.

Reply:
Ack, will do

Reply:
Done.